src/interfaces/coral_web/src/components/EditEnvVariablesButton.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useContext, useMemo, useState } from 'react';
+
+import {
+  BasicButton,
+  Button,
+  Dropdown,
+  DropdownOptionGroups,
+  Input,
+  Text,
+} from '@/components/Shared';
+import { ModalContext } from '@/context/ModalContext';
+import { useListAllDeployments } from '@/hooks/deployments';
+import { useUpdateDeploymentEnvVariables } from '@/hooks/envVariables';
+import { cn } from '@/utils';
+
+export const EditEnvVariablesButton: React.FC<{ className?: string }> = ({ className }) => {
+  const { open, close } = useContext(ModalContext);
+
+  const handleClick = () => {
+    open({
+      title: 'Edit .env variables',
+      content: <EditEnvVariablesModal onClose={close} />,
+    });
+  };
+
+  return (
+    <BasicButton
+      label="Edit .env variables"
+      size="sm"
+      kind="minimal"
+      className={cn('py-0', className)}
```

Comment:
Looks like we only have 1 instance of this component and the `className` value is `py-0` anyways
```suggestion
      className='py-0'
```